content/docs/reactivesearch/vue/search/SearchBox.md

```diff
@@ -1258,4 +1312,49 @@ The following events to the underlying `input` element:
 
 ## Examples
 
+### SearchBox with default props
+
 <a href="https://reactivesearch-vue-playground.netlify.com/?selectedKind=Search%20Components%2FSearchBox&selectedStory=Basic&full=0&addons=1&stories=1&panelRight=0" target="_blank">SearchBox with default props</a>
+
+
+### Customize suggestions using innerClass
+<img src="https://i.imgur.com/nmFY3Ha.png" style="margin:0 auto;display:block;"/>
+
+```jsx
+	<SearchBox
+	    title="SearchBox"
+	    dataField={['original_title', 'original_title.search']}
+	    componentId="BookSensor"
+	    innerClass={{
+            'section-label': 'section-label',
+	    	'active-suggestion-item': 'active-test-suggestion',
+	    	'suggestion-item': 'test-suggestion',
+	    }}
+	    enableFeaturedSuggestions
```

Comment:
@SavvyShah 

- `searchboxId` prop is missing here. (the change has to happen in React version also)
- Can we add a link to a featuredSuggestions example that we have?